Devices/Devices.py

```diff
@@ -781,10 +781,16 @@ def _bank_parameters(bank_index):
             #return []
 
     #return []
-    
-    
+
+
 def best_of_parameter_bank(device, device_bob_dict = DEVICE_BOB_DICT, ubermap_skip = False):
     """ Revised function by Stray that should work fine with any type of device. """
+    device_name = device.class_display_name
+
+    """"Fix for Massive that only works if the first 8 parameters are the macro knobs, which is default"""
+    if device_name == 'Massive':

```

Comment:
I wonder if there are other devices which might benefit from this fix also? In which case, it might make sense to add an array of `devices_with_macro_knobs` or whatever to `Devices.cfg`, then additional devices can be added just with a config change. I'm easy though as I don't have time to change/test anything myself at the minute :)